reference/sql/statements/admin.md

```diff
@@ -27,24 +27,32 @@ ADMIN SHOW DDL JOBS [NUM] [WHERE where_condition];
 * `NUM`: to view the last `NUM` results in the completed DDL job queue. If not specified, `NUM` is by default 10.
 * `WHERE`: to add filter conditions.
 
+To view the original SQL statements of the DDL job corresponding to `job_id`, use `ADMIN SHOW DDL JOB QUERIES`:
+
 {{< copyable "sql" >}}
 
 ```sql
 ADMIN SHOW DDL JOB QUERIES job_id [, job_id] ...;
 ```
 
+To cancel the running DDL job corresponding to `job_id`, use `ADMIN CANCEL DDL JOBS`. This statement returns the message that says whether the job has been cancelled.
+
 {{< copyable "sql" >}}
 
 ```sql
 ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
 ```
 
+To check the consistency of all the data and corresponding indexes in the `tbl_name` table, use `ADMIN CHECK TABLE`. If the consistency check is passed, an empty result is returned. On failure, return an error message says that data is inconsistent.
```

Comment:
```suggestion
To check the consistency of all the data and corresponding indexes in the `tbl_name` table, use `ADMIN CHECK TABLE`. If the consistency check is passed, an empty result is returned. Otherwise, an error message is returned indicating that the data is inconsistent.
```